### PR TITLE
Add little reminder that o-s-registrations wont be accepted if reg full

### LINKED ISSUE #3935: Clarify that on-the-spot registrations are not accepted when the competitor limit has been reached
The current way the registration requirements are phrased is misleading when there is a competitor limit and on-the-spot registrations are being accepted. It should be clarified that on-the-spot registrations will only be accepted when the competitor limit hasn't been reached, to prevent unregistered competitors from thinking that the competitor limit only applies to online registrations and that on-the-spot registrations do not apply to this limit.

See below for example.

screenshot 2019-02-20 08 17 12

I suggest that the line about on-the-spot registrations be changed to:  

*On-the-spot registrations will be accepted with a base registration fee of [cost] **if the competitor limit has not been reached by [online registration end date].***

(Side note: "On the spot" should be "On-the-spot" since it is a compound adjective that is directly in front of a noun).

### PR DESCRIPTION
**Pull request by ligio90**
Fixes #3935

### CODE REVIEW

File: WcaOnRails/config/locales/en.yml [hunk 1]
```diff
@@ -1020,7 +1020,7 @@ en:
       use_stripe_below_html: "The registration fee has to be paid through Stripe below once registered."
       refund_policy_html: "If your registration is cancelled before %{limit_date_and_time} you will be refunded a %{refund_policy_percent} of your registration fee."
       no_refunds: "Registration fees won't be refunded under any circumstance."
-      on_the_spot_registration: "On the spot registrations will be accepted with a base registration fee of %{on_the_spot_base_entry_fee}."
+      on_the_spot_registration: "On the spot registrations will be accepted with a base registration fee of %{on_the_spot_base_entry_fee} if the competitor limit has not been reached by %{end_of_registration}"
```
Comment (jfly) [inline WcaOnRails/config/locales/en.yml]: Could you include a screenshot of what this looks like? I think we tend to call `wca_local_time` before rendering datetimes to users. You can see an example of this here: https://github.com/jfly/worldcubeassociation.org/blob/00b66534978cd467eff5c0ecef0e12971deb2bcf/WcaOnRails/app/views/posts/_post.html.erb#L26

Comment (ligio90) [reply WcaOnRails/config/locales/en.yml]: image
`... end_of_registration: wca_local_time(@competition.registration_close))`
computer says no

Comment (ligio90) [reply WcaOnRails/config/locales/en.yml]: also, I'm sorry about the screenshot, I totally forgot!

image

**Comment by ligio90** [comment]
image
Also added formatting of the "fully paid" text, it used to say '...paid 20.00...' instead of '...paid $20 (United States Dollar)

File: WcaOnRails/config/locales/en.yml [hunk 2]
```diff
@@ -1024,7 +1024,7 @@ en:
       use_stripe_below_html: "The registration fee has to be paid through Stripe below once registered."
       refund_policy_html: "If your registration is cancelled before %{limit_date_and_time} you will be refunded a %{refund_policy_percent} of your registration fee."
       no_refunds: "Registration fees won't be refunded under any circumstance."
-      on_the_spot_registration: "On the spot registrations will be accepted with a base registration fee of %{on_the_spot_base_entry_fee}."
+      on_the_spot_registration_html: "On the spot registrations will be accepted with a base registration fee of %{on_the_spot_base_entry_fee} if the competitor limit has not been reached by %{end_of_registration}"
```
Comment (jfly) [inline WcaOnRails/config/locales/en.yml]: Missing a period at the end of the sentence =(

Comment (ligio90) [reply WcaOnRails/config/locales/en.yml]: =(

**Comment by jfly** [comment]
Thanks, @ligio90! :chart_with_upwards_trend:

**Pull request merged**